soap/src/java/com/zimbra/soap/account/message/HABGroup.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2018 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+package com.zimbra.soap.account.message;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import com.zimbra.common.soap.AccountConstants;
+import com.zimbra.soap.type.ZmBoolean;
+
+/**
+ * @author zimbra
+ *
+ */
+/**
+ * @zm-api-command-auth-required true
+ * @zm-api-command-description Returns a list of HABGroup and its children
+ */
+@XmlAccessorType(XmlAccessType.NONE)
+@XmlRootElement(name = AccountConstants.E_GET_HAB_RESPONSE)
+public class HABGroup {
+    /**
+     * @zm-api-field-description List of HabGroups under the root group
+     */
+    @XmlElement(name="name", required=false)
+    private String name;
+    
+    /**
+     * @zm-api-field-description List of HabGroups under the root group
+     */
+    @XmlElement(name="id", required=false)
+    private String id;
+    
+    /**
+     * @zm-api-field-description List of HabGroups under the root group
+     */
+    @XmlElement(name="habGroups", required=false)
+    private List<HABGroup> childGroups = new ArrayList<HABGroup>();
+    
+
+    private Map<String, Object>attrs = new HashMap<String,Object>();
```

Comment:
You need to treat this like other attrs.
See for example `src/java/com/zimbra/soap/account/message/GetInfoResponse.java`
Look for @ZimbraKeyValuePairs annotation